### PR TITLE
📦️ OPS-7902: Build appropriate image for hpc-api

### PR DESCRIPTION
**Pull request by s0**
See OPS-7902 for more details

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,14 @@
-# TODO: Amend for production 
+FROM public.ecr.aws/unocha/nodejs:12-alpine
```
Comment (teodorescuserban) [inline Dockerfile:1]: 👍

File: Dockerfile [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
 
-FROM unocha/nodejs:12
+ARG COMMIT_SHA
+ARG TREE_SHA
+ENV HPC_ACTIONS_COMMIT_SHA $COMMIT_SHA
+ENV HPC_ACTIONS_TREE_SHA $TREE_SHA
 
-RUN apk add -U build-base python3 py-pip
```
Comment (teodorescuserban) [inline Dockerfile:5]: 💯

**Review by unocha-hpc — APPROVED** [review]
Docker image has been successfully built and pushed as: `public.ecr.aws/unocha/hpc-api:v0.0.1-pre`

Please deploy this image to a development environment, and test it is working as expected before merging this pull request.

**Review by unocha-hpc — APPROVED** [review]
Docker image has been successfully built and pushed as: `public.ecr.aws/unocha/hpc-api:v0.0.1-pre`

Please deploy this image to a development environment, and test it is working as expected before merging this pull request.

**Review by unocha-hpc — APPROVED** [review]
Docker image has been successfully built and pushed as: `public.ecr.aws/unocha/hpc-api:v0.0.1-pre`

Please deploy this image to a development environment, and test it is working as expected before merging this pull request.

**Pull request merged**